frontend/packages/integration-tests-cypress/views/guided-tour.ts

```diff
@@ -1,5 +1,7 @@
 export const guidedTour = {
   close: () => {
+    // eslint-disable-next-line cypress/no-unnecessary-waiting
+    cy.wait(1000); // wait a sec for modal if it is going to be displayed
```

Comment:
No change required for the sake of this PR, but we should take a closer look at why we need a wait and conditional logic here.  The app should be in a known state, meaning we should know whether or not the modal is supposed to be present before we try to close it. See https://docs.cypress.io/guides/core-concepts/conditional-testing#Element-existence for more info and suggestions.

Reply:
Also, check out https://docs.cypress.io/guides/core-concepts/conditional-testing#Welcome-wizard as it's almost precisely this scenario.

Reply:
I suggest we open a bug to track removing the wait if possible.

Reply:
I was overthinking the cypress test I created for the new url query params.  The test runs succesfully even if Guided Tour modal is open.  For now I just removed the `cy.wait()`.   Long term I think we need to use [app.waitForLoad()](https://github.com/openshift/console/blob/1ef2f7e24430a74f945b6990270dedbcd507dada/frontend/packages/dev-console/integration-tests/support/pages/app.ts#L14) and switch perspectives & clode Guided Tour like dev console [does here](https://github.com/openshift/console/blob/1ef2f7e24430a74f945b6990270dedbcd507dada/frontend/packages/dev-console/integration-tests/support/pages/app.ts#L29).